sdk/communication/azure-communication-rooms/src/main/java/com/azure/communication/rooms/RoomsClient.java

```diff
@@ -457,4 +457,12 @@ private Map<String, ParticipantProperties> convertRoomIdentifiersToMapForRemove(
 
         return participantMap;
     }
+
+    private static String getUpdateRequest(Map<String, ParticipantProperties> participantMap) throws IOException {
```

Comment:
Should make this a shared method somewhere in implementation so that the sync and async clients don't need to define this themselves.

Reply:
moved to util/